diff --git a/src/component/shared.ts b/src/component/shared.ts
index dfa2d84102ebe5df1d4d4732d7d5fd5d5a079516..a7c6fcf1c634114cec9e6e879478df1cb3b044c5 100644
--- a/src/component/shared.ts
+++ b/src/component/shared.ts
@@ -70,4 +70,4 @@ export const DEFAULT_RETRY_BEHAVIOR: RetryBehavior = {
   base: 2,
 };
 // This ensures that the type satisfies the schema.
-const _ = {} as RetryBehavior satisfies Infer<typeof retryBehavior>;
+void ({} as RetryBehavior satisfies Infer<typeof retryBehavior>);
